<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>221</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnDivisibleByTwo">
       <property name="text">
        <string>Делимое на 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDivisibleByThree">
       <property name="text">
        <string>Делимое на 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDivisibleByFour">
       <property name="text">
        <string>Делимое на 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDivisibleByFive">
       <property name="text">
        <string>Делимое на 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDivisibleBySix">
       <property name="text">
        <string>Делимое на 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblResult">
       <property name="text">
        <string>Количество найденных чисел:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
